src/main/java/com/example/videotecha/mapper/MovieMapper.java

```diff
@@ -0,0 +1,16 @@
+package com.example.videotecha.mapper;
+
+import com.example.videotecha.dto.MovieDto;
+import com.example.videotecha.model.Movie;
+
+public class MovieMapper {
```

Comment:
If MovieMapper is a static-utility class, it's not a bad idea to mark it as `final` and hide the noargs constructor as `private`